keyboards/tn_alice/info.json

```
@@ -0,0 +1,129 @@
+{
+    "manufacturer": "Tony Studio",
+    "keyboard_name": "tn_alice",
+    "maintainer": "thong-phn",
+    "bootloader": "uf2boot",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true
+    },
+    "processor": "STM32F103",
+    "indicators": {
+        "caps_lock": "A2",
+        "on_state": 0      
+    },
+    "rgblight":{
+        "led_count": 22,
```

migrated from `config.h`
```suggestion
        "layers": {
            "enabled": true
        },
        "led_count": 22,
```